github/activity_notifications.go

```diff
@@ -178,6 +178,23 @@ func (s *ActivityService) MarkThreadRead(ctx context.Context, id string) (*Respo
 	return s.client.Do(ctx, req, nil)
 }
 
+// MarkThreadDone marks the specified thread as done.
+// Marking a thread as "done" is equivalent to marking a notification in your notification inbox on GitHub as done.
+//
+// GitHub API docs: https://docs.github.com/rest/activity/notifications#mark-a-thread-as-done
+//
+//meta:operation DELETE /notifications/threads/{thread_id}
+func (s *ActivityService) MarkThreadDone(ctx context.Context, id string) (*Response, error) {
```

Comment:
According to this: https://docs.github.com/en/rest/activity/notifications?apiVersion=2022-11-28#mark-a-thread-as-done
`id` is an integer, so let's please change this:

```suggestion
func (s *ActivityService) MarkThreadDone(ctx context.Context, id int64) (*Response, error) {
```

Reply:
Sounds good to me, I'll update the PR.  The MarkThreadRead function (which I shamelessly copied from) has the same issue, would you like me to fix it as part of this PR, or put in a separate PR to keep things clean?

Reply:
Yes, a separate PR would be great since that will be a breaking change. Thank you!